plugins/connection/network_cli.py

```diff
@@ -1225,7 +1225,7 @@ def get_file(self, source=None, destination=None, proto="scp", timeout=30):
         """Fetch file over scp/sftp from remote device"""
         ssh = self.ssh_type_conn._connect_uncached()
         if self._ssh_type == "libssh":
```

Comment:
wouldn't this be `"ansible.netcommon.libssh"` for libssh ? as set here
https://github.com/ansible-collections/ansible.netcommon/blob/c40126937cd20edd9222b85a8832b1b79c611107/plugins/connection/network_cli.py#L456

Reply:
This might actually be a larger issue, but I've worked around it here and I've opened https://github.com/ansible-collections/ansible.netcommon/issues/282 to get back to the issue